lib/src/server/server.dart

```diff
@@ -257,7 +257,8 @@ class DevServer {
             defaultDocument: _entryPath));
     await shelf.serve(handler, host, port);
     print('Serving $_entryPath at http://$host:$port/');
-    compiler.run();
+    CheckerResults results = compiler.run();
```

Comment:
nice catch. I think this was supposed to return Future<CheckerResults> but I think CheckerResults is deprecated at this point, so +1 just returning status.